requirements.txt

```diff
@@ -1,7 +1,27 @@
-django==4.1
+aiohttp==3.9.1
```

Comment:
[This comment](https://github.com/mate-academy/py-images-and-custom-auth-serializer/pull/403#discussion_r1431190545) is not resolved, remove the `media/uploads/movies` folder with files from the pull request.